packages/@react-spectrum/card/stories/HorizontalCard.stories.tsx

```diff
@@ -10,144 +10,135 @@
-const StoryFn = ({storyFn}) => storyFn();
-
-const meta: Meta<SpectrumCardProps> = {
+export default {
   title: 'Card/horizontal',
-  component: Card,
-  decorators: [storyFn => <StoryFn storyFn={storyFn} />]
+  component: Card
+} as ComponentMeta<typeof Card>;
+
+export const CardGrid: CardStory = {
+  render: (args, context) => <Card {...args} {...context} />,
+  args: {...Horizontal.args, layout: 'grid'},
+  decorators: [(Story, context) => (
+    <CardGridDecorator
+      Story={Story}
+      context={context} />
+  )]
 };
 
-export default meta;
-
-
-const Template = (): Story<SpectrumCardProps> => (args) => (
-  <div style={{height: '90px'}}>
-    <Card {...args} />
-  </div>
-);
-
-export const Horizontal = Template().bind({});
-Horizontal.args = {...Default.args, orientation: 'horizontal'};
-
-export const HorizontalSquare = Template().bind({});
-HorizontalSquare.args = {...Horizontal.args, ...DefaultSquare.args};
-
-export const HorizontalTall = Template().bind({});
-HorizontalTall.args = {...Horizontal.args, ...DefaultTall.args};
-
-export const HorizontalNoDescription = Template().bind({});
-HorizontalNoDescription.args = {...Horizontal.args, ...NoDescription.args};
-
-export const HorizontalNoDescriptionSquare = Template().bind({});
-HorizontalNoDescriptionSquare.args = {...Horizontal.args, ...NoDescriptionSquare.args};
-
-export const HorizontalWithIllustration = Template().bind({});
-HorizontalWithIllustration.args = {...Horizontal.args, ...WithIllustration.args};
-
-export const HorizontalLongTitle = Template().bind({});
-HorizontalLongTitle.args = {...Horizontal.args, ...LongTitle.args};
-
-export const HorizontalLongDescription = Template().bind({});
-HorizontalLongDescription.args = {...Horizontal.args, ...LongContent.args};
+const CardGridDecorator = (props) => {
+  let {Story, context} = props;
+  let {args} = context;
+  return (
+    <div
+      style={{
+        width: '100%',
+        margin: '50px',
+        display: 'grid',
+        gap: '20px',
+        gridTemplateColumns: 'repeat(auto-fit, 360px)',
+        gridAutoRows: '104px'
+      }}>
+      {
+        (new Array(15).fill(0)).map((_, index) => {
+          let url = getImage(index);
+          return (
+            <Story {...args} key={`${index}${url}`}>
+              <Image src={url} />
+              <Heading>Title {index}</Heading>
+              <Text slot="detail">PNG</Text>
+              <Content>Description</Content>
+            </Story>
+          );
+        })
+      }
+    </div>
+  );
+};
 
-export const HorizontalLongDetail = Template().bind({});
-HorizontalLongDetail.args = {...Horizontal.args, ...LongDetail.args};
+export const CardFloat: CardStory = {
+  render: (args, context) => <Card {...args} {...context} />,
+  args: {...Horizontal.args},
+  decorators: [(Story, context) => (
+    <CardFloatDecorator
+      Story={Story}
+      context={context} />
+  )]
+};
```

Comment:
You may have alluded to think in our slack convo but just to double check and for others:
So theoretically would you be able to import the `CardFloat` story from `Card.stories.tsx` and just provide it with the `Horizontal.args` since it has the same overall setup? No need to update the PR if so, just curious if I'm thinking about the decorators and story definitions in the right way.

Reply:
Similar possible situation with the `QuietCard` stories borrowing from `Card` stories

Reply:
Yeah, i'm not sure why some of them differ so much between horizontal/quiet/standard, but theoretically we could condense these more if those could line up better